src/aig-common/modules/standard/components/cpv-new-form/cpv-new-form.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { FuseProgressBarService } from '@fuse/components/progress-bar/progress-bar.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { CpvResourceService, CpvDTO } from 'aig-standard';
+import { EventService } from 'aig-common/event-manager/event.service';
+
+@Component({
+    selector: 'aig-cpv-new-form',
+    templateUrl: './cpv-new-form.component.html',
+    styleUrls: ['./cpv-new-form.component.scss']
+})
+export class AigCpvNewFormComponent implements OnInit {
+    step: any = {
+        form: true,
+        loading: false,
+        complete: false
+    };
+
+    constructor(
+        private _formBuilder: FormBuilder,
+        private _fuseProgressBarService: FuseProgressBarService,
+        private _snackBar: MatSnackBar,
+        private cpvResourceService: CpvResourceService,
+        private eventService: EventService,
+    ) { }
+
+    cpvCategorys = ['GENERIC', 'PERSON'];
```

Comment:
for cpv not is necessary, categoy. For understand the fileds of cpv you can check in CpvDTO interface